package.json

```diff
@@ -120,21 +120,8 @@
     "*.js": "eslint"
   },
   "release": {
-    "plugins": [
```

Comment:
you are really removing a lot :D why is that?

Reply:
It's the default config already, so this cleans it up